PORTAL/jobs/_utils.py

```diff
@@ -477,7 +478,11 @@ def get_utc_datetime(
             timestamp = datetime.datetime.fromisoformat(timestamp)
             timestamp = timestamp.astimezone(tzinfo)
         else:
-            m = re.match(r'(\d{4}-\d\d-\d\d(.)\d\d:\d\d:\d\d)(\.\d{3}(?:\d{3})?)?([+-]\d\d:?\d\d.*)?', timestamp)
+            m = re.match(
+                r'(\d{4}-\d\d-\d\d(.)\d\d:\d\d:\d\d)'
+                r'(\.\d{3}(?:\d{3})?)?([+-]\d\d:?\d\d.*)?',
+                timestamp
+            )
```

Comment:
If we're splitting this up anyway, let's make it a verbose regex:

```suggestion
            m = re.match(r'''
                (
                    \d{4} - \d\d - \d\d
                    (.)  # <sep>
                    \d\d : \d\d : \d\d
                 )  # <body>
                ( \. \d{3} (?: \d{3} )? )?  # <subzero>
                ( [+-] \d\d :? \d\d .* )?  # <tz>
                ''',
                timestamp,
                re.VERBOSE,
            )
```